Trappist/src/Promact.Trappist.Repository/TestDashBoard/TestDashBoardRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Promact.Trappist.DomainModel.DbContext;
+using Promact.Trappist.DomainModel.Models.Test;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Promact.Trappist.Repository.TestDashBoard
+{
+    public class TestDashBoardRepository : ITestDashBoardRepository
+    {
+
+        private readonly TrappistDbContext _trappistdb;
+        public List<Test> Tests;
+
+        public TestDashBoardRepository(TrappistDbContext trappistdb)
+        {
+            _trappistdb = trappistdb;
+
+
+        }
+
+        /// <summary>
+        /// Fetch all the tests from Test Model,Convert it into List
+        /// </summary>
+        /// <returns>List of Tests</returns>
+        public List<Test> GetAllTests()
+        {
+                      
+            var Tests = _trappistdb.Test.ToList();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Rename 'Tests' which hides the field with the same name. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1117)